### DISCUSSION TITLE
[Package]: Best browser in repo

### DISCUSSION THREAD

**Discussion by anshiq** [termux/termux-packages#10227] [Q&A] [answered]
### Package description

Can anybody tell me which is the best browser in termux repo I can use for maximum tasks. Like fire fox or chromium.

### Home page URL

Termux.com

### Source code URL

Termux.com

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/x11-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

No

**Comment by ghost** [comment, marked as answer]
We only have otter browser and netsurf, both of these browsers doesn't have much features compared to chromium and lower html5 scoring. netsurf only loads barebones of webpage and no JavaScript support compared to otter browser while otter browser renders most websites but it's webkit engine is obsolete and outdated and much like IE (likely it will ask you to upgrade browser), so some websites may or may not work as expected so don't expect your "maximum task" meets your requirements

In short: otter browser is so far the best browser to load most sites including modern ones but with a little broken elements 

**Edit: we have surf** as the third browser with highest html score than others but it's very minimalist to use

**Comment by T-Dynamos** [comment]
I am trying compiling chromium

**Reply by anshiq** [reply]
bro do you still compilling chromiun, Can you need help.

**Comment by thiled** [comment]


**Reply by macerg** [reply]
I suggest install Andronix app and from there, install a flavour of Linux within Termux. I have Debian buster installed. Comes with a complete desktop environment. Install Firefox or Chromium there.

Explanation:- A Linux implementation via the Andronix app uses proot within Termux to run a full-blown Linux within Termux. It's wonderful! Once you have it up and running, the actual Andronix app can be uninstalled from your phone as it is only used for the initial system setup.

**Reply by macerg** [reply]
Or proot-distro and then install your preferred flavour of Linux.
I have Debian bullseye installed. It's less than 2GB with Firefox installed.

**Comment by ghost** [comment]
We have [surf](https://github.com/termux/termux-packages/tree/master/x11-packages/surf) packaged and it has a higher html5 score than other browsers which uses Apple's webkit (webkitgtk) so this is the so far the third and first browser in Termux to have highest html5 score so many websites may work
